.ci/ci_job_flags.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2020 Intel Corporation
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script should be sourced so that the environment gets exported variables
+# used to control the setup and execution of tests.
+#
+# Export the CI_JOB environment variable if you need job-specific variables
+# set apart from the defaults.
+#
+
+CI_JOB=${CI_JOB:-}
+
+# List of all setup flags used by scripts
```

Comment:
What about call this `init_ci_flags_defaults` and adding to the comment that many of these defaults are updated depending on the job type?